### PR TITLE
feat: add table is-dark class similar to the container

### PR DESCRIPTION
**Pull request by guastallaigor**
This address #77 if this feature request is approved.

image

image

### CODE REVIEW

**Comment by BcRikko** [comment]
I'm sorry I let you fix it over and over 🙇

**Comment by guastallaigor** [comment]
Not sure if this is exactly the same as you requested, or if the code itself is good enough.

image

File: README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
   <a href="README.md">English</a> / <a href="README-jp.md">日本語</a>
 </div>
 
+[![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
```
[the remark this answers is not in the document]
Comment (guastallaigor) [reply README.md]: Sorry about that

**Comment by evexoio** [comment]
this PR's commit history needs god 😂
can someone fix it?

**Comment by guastallaigor** [comment]
@evexoio You're right, I'm sorry, I forgot about that. I will try my best when I have some more free time today.

**Comment by guastallaigor** [comment]
I could not find a way to fix this yet. This commit history problem happened in #76 too.

If any of you has an idea to fix this or something to point me to the right direction that would be great.

I notice this PR is also missing the storybook, so I will try to add this.

**Comment by evexoio** [comment]
@guastallaigor you can use cherry-picking to solve that, google it and you'll find plenty of resources about how it works.
Let's talk on gitter if you want me to help with it :)

**Comment by trezy** [comment]
Hey @guastallaigor, make sure to merge the latest version of the `develop` branch so that our CI will run the required checks for this PR. 😉

**Comment by guastallaigor** [comment]
@trezy Sorry about that, I guess my fork wasn't up to date with this repo.
With a great help from @evexoio the commit history is clear now.
I also added the `is-dark` to storybook.
So now everything with this PR should be good.
But if I forgot anything else, please let me know.
Thank you all for your patience.

**Comment by guastallaigor** [comment]
Wait a minute the storybook is still wrong. I will look into it.

The style in `is-dark` class is old and different from what it is now. Compare your image to this:

image

**Comment by guastallaigor** [comment]
I didn't do the commit history clear right so I lost some of the style, like box-shadow and background-linear. But I code it again and now it should be good :+1: 

I just don't remember if it's the same as before, so if you guys want to review it again that's ok :)

**index.html**

image

**storybook**

image

File: scss/elements/tables.scss [hunk 2]
```diff
@@ -16,14 +10,57 @@
...
+      left: 0;
+      background: linear-gradient(to left, transparent 4px, #fff 4px) no-repeat,
+        linear-gradient(to bottom, transparent 4px, #fff 4px) no-repeat,
+        linear-gradient(to right, transparent 4px, #fff 4px) no-repeat,
+        linear-gradient(to top, transparent 4px, #fff 4px) no-repeat;
+      background-position: 100% 0, 100% 0, 0 100%, 0 100%;
+      background-size: calc(100% - 4px) 4px, 4px calc(100% - 4px);
+    }
+
+    @include thsAndTdsBoxShadow(#fff);
```
Comment (BcRikko) [inline scss/elements/tables.scss]: could you use a variable? 🙏 
```suggestion
    @include thsAndTdsBoxShadow($background-color);
```

File: scss/elements/tables.scss [hunk 3]
```diff
@@ -16,14 +10,57 @@
...
+      content: "";
+    }
+
+    &::after {
+      top: 0;
+      right: 0;
+      bottom: 0;
+      left: 0;
+      z-index: 1;
+    }
```
Comment (BcRikko) [inline scss/elements/tables.scss]: `&::after` does not seem to be used 🤔 

```diff
-    &::before,
-    &::after {
-      position: absolute;
-      content: "";
-    }

-    &::after {
-      top: 0;
-      right: 0;
-      bottom: 0;
-      left: 0;
-      z-index: 1;
-    }

  &::before {
+   position: absolute;
+   content: "";
    top: 0;
     ....
  }
```

File: scss/elements/tables.scss [hunk 4]
```diff
@@ -16,14 +10,57 @@
...
+
+    &::before {
+      top: 0;
+      right: 0;
+      bottom: 0;
+      left: 0;
+      background: linear-gradient(to left, transparent 4px, #fff 4px) no-repeat,
+        linear-gradient(to bottom, transparent 4px, #fff 4px) no-repeat,
+        linear-gradient(to right, transparent 4px, #fff 4px) no-repeat,
+        linear-gradient(to top, transparent 4px, #fff 4px) no-repeat;
```
Comment (BcRikko) [inline scss/elements/tables.scss]: same here 🙇 
`$background-color`

File: scss/elements/tables.scss (hunk 3 above)
Comment (guastallaigor) [reply scss/elements/tables.scss]: You're right, removed.

File: scss/elements/tables.scss (hunk 2 above)
Comment (guastallaigor) [reply scss/elements/tables.scss]: Changed all `#fff` to `$background-color`.

File: scss/elements/tables.scss (hunk 4 above)
Comment (guastallaigor) [reply scss/elements/tables.scss]: Changed all `#fff` to `$background-color`.

**Pull request merged**